BUILDING.md

```diff
@@ -110,17 +125,28 @@ platforms. This is true regardless of entries in the table below.
 
 Depending on the host platform, the selection of toolchains may vary.
 
-#### Unix
-
-* GCC 4.9.4 or newer
-* Clang 3.4.2 or newer
-
-#### AIX
-* GCC 6.3 or newer
-
-#### Windows
-
-* Visual Studio 2017 with the Windows 10 SDK on a 64-bit host.
+| Operating System | Compiler Versions                                           |
+| ---------------- | ----------------------------------------------------------- |
+| Linux            | GCC >=6.3, Clang >=3.4.2                                    |
```

Comment:
I don't know what version of Clang should be our minimum so it's unchanged, anyone got input here? @bnoordhuis @ofrobots @refack?

Reply:
Either remove as we don't test that at all. Or CLang-8 which is the latest stable, and AFAIK is libstd++ independent anyway.

Reply:
Removed entirely, perhaps a project for the future.